index.html

```diff
@@ -82,8 +85,13 @@
     <section id="sotd">
       <p>
         This document is <strong>work in progress</strong> and is subject to
-        change. The API is being developed based on requirements from the TV
-        Control Working Group, and previously the TV Control Community Group.
+        change. The API has significantly evolved since publication as a First
+        Public Working Draft on
+        <a href="https://www.w3.org/TR/2016/WD-tvcontrol-api-20160915/"
+        title="First Public Working Draft of the TV Control API">15
+        September 2016</a>. The <code>TVTuner</code> class has been removed
+        in particular, and the API follows a more flexible source-centric
+        design.
```

Comment:
I had already published the updated WD to /TR/ space, so I took the liberty to push the SOTD update directly to be able to re-publish the WD right away, see: https://github.com/w3c/tvcontrol-api/commit/b94e2b476cb73e13299853d15e363e8f71183a1e